test/gateway/eip20_gateway/constructor.js

```diff
@@ -150,6 +157,22 @@ contract('EIP20Gateway.constructor() ', function (accounts) {
         );
 
     });
+    
+    it('should fail when members manager address is passed as zero', async function () {
```

Comment:
This looks like it belongs to the `Organized` tests and not the `EIP20Gateway` tests. If we change `Organized` and make the members manager optional, now we have to update test cases for the EIP-20 Gateway, which we didn't actually touch.

Reply:
I think it's fine to keep this test. 
Because there is no other way to test that child contract calls the constructor of the parent.

Reply:
For constructors and overridden functions, I think we should have the test.

Reply:
OK if you want, but it is not correct to have this test here.